pkg/admission/plugins.go

```diff
@@ -45,6 +46,7 @@ func GetPlugins() []string {
 	for k := range plugins {
 		keys = append(keys, k)
 	}
+	sort.Strings(keys)
```

Comment:
Was this supposed to be in this PR?  Is so what does it do?  Make the generated output stable?